src/DIRAC/WorkloadManagementSystem/Service/WMSAdministratorHandler.py

```diff
@@ -217,7 +217,7 @@ def export_getJobPilotOutput(self, jobID):
             c = cycle
 
     if pilotReference:
-      return getGridJobOutput(pilotReference)
+      return getGridJobOutput(pilotReference.decode())
```

Comment:
This should be fixed a level deeper in one or more of: `getAtticJobParameters`/`getJobParameter`/`getJobParameters`

Reply:
@TaykYoku has debugged this further and got to the bottom of it on slack: https://lhcbdirac.slack.com/archives/C3AGWCA8J/p1625147097062700

In summary, the JobParameters table is using `BLOB` for the value when it should be using a text type.